sjb/config/common/test_cases/origin_release_install_azure_39.yml

```diff
@@ -163,6 +163,34 @@ extensions:
           -e "openshift_azure_input_image_prefix=centos7" \
           playbooks/azure/openshift-cluster/launch.yml
 
+    - type: "script"
+      title: "run the acs-engine e2e test"
+      script: |-
+        git clone -b custom-images-e2e https://github.com/kargakis/acs-engine.git
```

Comment:
think you should add `kargakis,acs-engine=custom-images-e2e` to the sync stanza at the top of the file instead of this